src/intervals/precision.jl

```diff
@@ -1,83 +0,0 @@
-# This file is part of the IntervalArithmetic.jl package; MIT licensed
```

Comment:
I noticed that this file was deleted, and the functionality is not included elsewhere. Can you simply comment?

Reply:
Removal of all that is an open issue:

https://github.com/JuliaIntervals/IntervalArithmetic.jl/issues/228

IIRC this issue resulted from discussion on other issues, so the history of it is a bit more involved but as far as I remember there was a consensus about the removal.